packages/docs/pages/Alert/AlertPage.tsx

```diff
@@ -69,27 +69,44 @@ const AlertPage = () => (
       <Code>warning</Code>, <Code>success</Code>, <Code>info</Code>.
     </Text>
 
-    <Text>To use this component, wrap your app with this component:</Text>
+    <Text>
+      To use this component, put it in your root component (e.g. place it after <Code>GlobalStyles</Code> component):
+    </Text>
 
     <CodeSnippet>
-      {/* jsx-to-string:start */}
-      {function App() {
-        return (
-          <>
-            {/* ... */}
-            <AlertsManager manager={alertsManager} />
-            {/* ... */}
-          </>
-        );
-      }}
-
-      {/* jsx-to-string:end */}
+      {`
+        export const alertsManager = createAlertsManager(); // import this in child components to use alerts
+        
+        function App() {
+          return (
+            <>
+              {/* ... */}
+              <GlobalStyles />
+              <AlertsManager manager={alertsManager} />
+              {/* ... */}
+            </>
+          );
+        }
+      `}
     </CodeSnippet>
 
     <Text>
-      This works in conjunction with an instance created by <Code>createAlertsManager</Code> function below.
+      This works in conjunction with an instance created by <Code>createAlertsManager</Code> function below. You need to
+      export <Code>alertsManager</Code> instance and then you can import it in child components in order to trigger
+      alert:
     </Text>
 
+    <CodeSnippet>
```

Comment:
Maybe we should add an example on how to create a `useAlerts` hook.

Reply:
Agree it's a good idea, but I need to play with hook to become familiar with it, and unfortunately I'm not sure If I will have time for that in this sprint.